scripts/kind-e2e/README.md

```diff
@@ -86,6 +86,19 @@ manually, then run:
 make reload-images restart=none
 ```
 
+#### Re-running e2e tests after your code changes
+You can run the e2e tests without re-deploying everything by using the e2e-tests target.
+```bash
+make e2e-test
+```
+
+And you can focus the e2e tests on specific tags by using the focus makefile env. The following
+example runs only the tests tagged as [gateway].
```

Comment:
They're actually tagged as _[redundancy]_, not _[gateway]_ 😄

Reply:
I was referencing the new tests I was writing but let's use redundancy instead as as we talked about, they could end up being the same..